packages/analytics/analytics-chart/src/components/SimpleChart.cy.ts

```diff
@@ -23,17 +21,17 @@ const exploreResultTruncated = {
     },
   ],
   meta: {
-    start: 1685452140.987,
-    end: 1685473740.987,
-    queryId: '12346',
-    dimensions: {
-      StatusCode: ['200', '300'],
+    start_ms: 1685452140.987,
+    end_ms: 1685473740.987,
+    query_id: '12346',
+    display: {
+      StatusCode: { 200: { name: '200' }, 300: { name: '300' } },
```

Comment:
That's fine imo, it's just a component test, the component does't really care if it gets explore v4 dimensions or not, the structure is all that matters, this will still get rendered just fine